src/frontend/src/redux/records/types.ts

```diff
@@ -22,18 +8,15 @@ export interface Record {
   caseBalance: string;
 }
 
-export interface RecordsState {
+export const LOAD_SEARCH_RECORDS = 'LOAD_SEARCH_RECORDS';
+
+export interface SearchRecordState {
   records: Record[];
 }
 
-// These constants are used as the 'type' field in Redux actions.
```

Comment:
Why'd you get rid of this comment?

Reply:
Originally I made another types.ts file since I wasn't sure if the original file was intended for what I was doing, and I didn't want to alter it.  I then replaced the original file with what we currently have, and overlooked the comments. Comments are back in.